### PR TITLE
Fateme

### PR DESCRIPTION
**Pull request by fatemeebrahimzadeh**
include some changes that commented

### CODE REVIEW

File: fateme/challenge-1.js [hunk 1]
```diff
@@ -17,8 +17,12 @@ function isPrime(n) {
   return true;
 }
 
-// test case for prime numbers
+function isTypeValid(value, trueType) {
+  if (typeof value === trueType) return true;
```
Comment (Hossein13M) [inline fateme/challenge-1.js]: You don't need an `if - else` statement for this. You can use the following shorthand:
`return typeof value === trueType`

File: fateme/challenge-6.js [hunk 2]
```diff
@@ -1,15 +1,25 @@
 function pyramid(n = 5) {
-  for (let index = 1; index <= n; index++) {
-    const line = []
-    for (let j = 1; j <= n - index; j++) {
-      line.push(" ")
+  if (isTypeValid(n, "number")) {
```
Comment (Hossein13M) [inline fateme/challenge-6.js]: This approach is great but for more complicated types (which we will cover in the TypeScript course).

**Review by Hossein13M — APPROVED** [review]
Done

File: fateme/challenge-6.js (hunk 2 above)
Comment (fatemeebrahimzadeh) [reply fateme/challenge-6.js]: I remove type checking at all, hope to know what you mean

**Review by Hossein13M — APPROVED** [review]
done

**Pull request merged**